server/form-pages/apply/access-and-healthcare/index.ts

```diff
@@ -1,11 +1,20 @@
+// eslint-disable-next-line max-classes-per-file
 import AccessNeeds from './accessNeeds'
 import AccessNeedsMobility from './accessNeedsMobility'
 import Covid from './covid'
 import AccessNeedsAdditionalAdjustments from './accessNeedsAdditionalAdjustments'
+import { Task } from '../../utils/decorators'
 
 export default {
```

Comment:
It tripped me up that this doesn't follow the conventions of other files in this commit. Worth changing to `const pages = ....` and `export default pages`?

Reply:
Ah, we don't even need the page exports anymore! Will remove then